### DISCUSSION TITLE
Whats the right way of updating?

### DISCUSSION THREAD

**Discussion by BorisBrock** [cryinkfly/Autodesk-Fusion-360-for-Linux#345] [Q&A]
When starting Fusion360, a dialog pops up, telling me a new Version is available and that I should update soon.

What is the correct way of performing an update?
Simply running the installation again and reinstalling Fusion360?
Or is there a better/faster way?

**Comment by cryinkfly** [comment]
Currently the fastest option is to rename the old Wineprefix "$HOME/.fusion360/wineprefixes/default" to "$HOME/.fusion360/wineprefixes/default-backup" and then repeat the installation. ;)
